cli/flox-activations/src/cli/start_or_attach.rs

```diff
@@ -155,7 +152,7 @@ fn check_for_activation_ready_and_attach_pid(
     };
 
     let activation = activations
-        .activation_for_id_mut(activation_id)
+        .activation_for_store_path_mut(store_path)
```

Comment:
Another TODO, I think we can fall back to calling `StartOrAttach::handle` in this error case as well